fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/FromJoin.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * (C) Copyright IBM Corp. 2019
```

Comment:
```suggestion
 * (C) Copyright IBM Corp. 2021
```

Reply:
Thanks....still need to do a bunch of code cleanup, so not ready for review yet.

Reply:
done